hack/verify-golint.sh

```diff
@@ -10,45 +11,21 @@ if ! which golint &>/dev/null; then
 fi
 
 GO_VERSION=($(go version))
-echo "Detected go version: $(go version)"
 
-if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.4|go1.5') ]]; then
-  echo "Unknown go version '${GO_VERSION}', skipping golint."
+if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.[4-6]') ]]; then
+  echo "Unknown go version '${GO_VERSION[@]:2}', skipping golint."
   exit 0
 fi
 
 STI_ROOT=$(dirname "${BASH_SOURCE}")/..
-source "${STI_ROOT}/hack/common.sh"
+source "${STI_ROOT}/hack/util.sh"
 
 cd "${STI_ROOT}"
 
-arg="${1:-""}"
-bad_files=""
-
-if [ "$arg" == "-m" ]; then
-  head=$(git rev-parse --short HEAD | xargs echo -n)
-  bad_files=$(git diff-tree --no-commit-id --name-only -r master..$head | \
-    grep "^pkg" | grep ".go$" | grep -v "bindata.go$" | grep -v "Godeps" | \
-    grep -v "third_party" | xargs golint)
-else
-  find_files() {
-    find . -not \( \
-      \( \
-        -wholename './Godeps' \
-        -o -wholename './release' \
-        -o -wholename './target' \
-        -o -wholename './test' \
-        -o -wholename '*/Godeps/*' \
-        -o -wholename '*/third_party/*' \
-        -o -wholename '*/_output/*' \
-      \) -prune \
-    \) -name '*.go' | sort -u | sed 's/^.{2}//' | xargs -n1 printf "${GOPATH}/src/${STI_GO_PACKAGE}/%s\n"
-  }
-  bad_files=$(find_files | xargs -n1 golint)
-fi
+bad_files=$(s2i::util::find_files | xargs -n1 golint)
```

Comment:
Yes, before getting to a single implementation I had it changed in each file to something that makes sense.
`-wholename` is documented as less portable than `-path`, and we don't need all those rules (probably there just because it was copied from origin at some point).